### ISSUE TITLE
Cannot Edit HTML Content - Building Changes Nothing

### ISSUE THREAD

**Issue by Bluejamathons** [rr-/szurubooru#678]
Hello all, I've been having issues concerning editing certain pages of the site. I've followed every instruction I can find, including:

- Editing the desired `.tpl` files in the `/root/szuru/client/html` directory
- Editing docker-compose.yml so that the lines `image: szurubooru/server:latest` and `image: szurubooru/client:latest` are replaced with `build: server` and `build: client` respectively. I have also tried `build: ./server` and `build: ./client` as suggested [in this post](https://github.com/rr-/szurubooru/issues/484#issuecomment-1119817547)
- Building with `--no-cache`
- Trying to run the command `export DOCKER_BUILDKIT=1; export COMPOSE_DOCKER_CLI_BUILD=1`, however I'm not sure if it did anything as it didn't output anything in the terminal. I even tried running `export DOCKER_BUILDKIT=1` and `export COMPOSE_DOCKER_CLI_BUILD=1` as separate commands; they did the same amount of nothing.
- Editing `Dockerfile` in the `/root/szuru/client/` directory so that the lines `FROM --platform=$BUILDPLATFORM node:lts as builder` and `FROM --platform=$BUILDPLATFORM scratch as approot` are replaced by the following combinations: `<completely deleted>, <completely deleted>`, and `FROM --platform=$BUILDPLATFORM node:lts AS builder, FROM --platform=$BUILDPLATFORM scratch AS approot`, and `FROM node:lts as builder, FROM scratch as approot`, and `FROM node:lts AS builder, FROM scratch AS approot`.
- Starting szurubooru with the command `docker-compose start -d client` as suggested [in this post](https://github.com/rr-/szurubooru/issues/512#issuecomment-1287892075)

I believe that should be everything I've tried? I might have forgotten to mention something, or the answer is so obvious that I completely overlooked it or some BS lol. I've basically scoured every issue for anything that might remotely be related to my issues though

I am using Docker v27.1.2 and Docker-Compose v1.29.2
I have tried every combination of the above methods
The site is proxied through nginx
It works completely fine otherwise - I've never had any major issues getting it to start up.
I'm not sure if this will get any comments or any heed, but oh well. All I really wanted to do was add a Discord server link and this seems like an oddly difficult task for what should have taken all of five minutes to do.
To anyone who comments or helps in any way; thank you!

**Comment by Bluejamathons** [comment]
...for some reason it fixed itself after I decided I had enough and I turned my computer off lmao

I'm not quite sure what happened

**Comment by Bluejamathons** [comment]
if anyone has similar issues, I figured it out:

- I commented the lines `image: szurubooru/server:latest` and `image: szurubooru/client:latest` and replaced them with `build: server` and `build: client` respectively
- I replaced `FROM --platform=$BUILDPLATFORM node:lts as builder` and `FROM --platform=$BUILDPLATFORM scratch as approot` with `FROM node:lts AS builder` and `FROM scratch AS approot` respectively
- I ran `docker-compose down`
- I ran `docker-compose build --no-cache`
- I ran `docker-compose up -d`
- Everything up to this point has been listed somewhere, but I *then* had to restart my computer. Why? I have no idea. The site is being hosted on a separate server entirely. My computer has no connection except through SSH. Not sure if I'm oblivious and missed this detail somewhere or if this is entirely unexpected. Thought I'd clarify though just to be a good forum member
